test/artillery/photos-test.yaml

```diff
@@ -1,5 +1,5 @@
 config:
-  target: "http://localhost:6001/buildings/00000"
+  target: "http://localhost:6001/buildings/czzzu"
```

Comment:
Maybe it's arbitrary but is that parameter doing anything? Do you have a default for when a parameter like that hits your API?